tests/ci.sh

```diff
@@ -42,6 +46,13 @@ if [ "$MISMATCHER" = "True" ]; then
     exit 0
 fi
 
+#########################
+# Next branch tracking  #
+#########################
+if [ "$TEST_NEXT_BRANCH_TRACKING" ]; then
+    echo "Track next branches for parallaxsecond repositories"
+    python3 $(pwd)/parsec/utils/release_tracking.py $(pwd)/Cargo.toml
+fi
```

Comment:
Is it possible to print out the new toml for debug purposes?

Reply:
We print the diff of the toml [in the release_tracking.py script](https://github.com/parallaxsecond/parsec/blob/main/utils/release_tracking.py#L42) as you can see [here](https://github.com/tgonzalezorlandoarm/parsec-tool/actions/runs/7020560449/job/19100580803#step:4:212)